forge/lang/expander.rkt

```diff
@@ -309,8 +309,8 @@
               prop-expr:BlockClass
               (~optional -scope:ScopeClass)
               (~optional -bounds:BoundsClass)
-              "where"
-              where-blocks:TestConstructClass ...)
+              (~optional (~seq where-blocks:TestConstructClass ...))
```

Comment:
The `~optional ~seq` can go away because the parser has `/WHERE-TOK` instead of `WHERE-TOK`. Nice!